src/telemetry/typings/vscode-extension-telemetry.d.ts

```diff
@@ -6,6 +6,11 @@ declare module 'vscode-extension-telemetry' {
       properties?: { [key: string]: string },
       measures?: { [key: string]: number }
     ): void;
+    sendTelemetryException(
```

Comment:
Huh.  You woulda thought the telemetry module would have the needed typings.

Reply:
This is a result of using a wrapper class. Once we move it to it's own object we won't have to do this